_posts/2020-07-24-moveit-research-roundup.md

```diff
@@ -0,0 +1,52 @@
+---
+author: Mark Moll
+comments: false
+date: 2020-07-24
+layout: post
+title: 'MoveIt Research Roundup'
+media_type: image
+media_link: /assets/images/blog_posts/moveit_research/FetchMagicLeap.jpg
+description: A brief selection of robotics research that was enabled by MoveIt
+categories:
+- MoveIt
+- ROS
+---
+
+This is the first in a series of posts where we will highlight interesting robotics research that leverages MoveIt in the experimental evaluation. Most of the research results may not be immediately available within MoveIt itself, but we expect that as the underlying ideas mature they will make their way into MoveIt in some shape or form. We highly encourage the original authors to be active participants in this process, not only as a service to the robotics community, but also to increase the impact of the research: if research is easy to replicate, it is more likely to be adopted and further improved.
+
+If you'd like to see your work featured here, feel free to email [Mark Moll](mailto:mark@picknik.ai).
+
+## A New Reality
+
+Augmented reality (AR) is increasingly being explored as a way to interact with robots. Communication via AR can work in both directions: it can be an input modality for a user to manipulate a virtual copy of a robot or some object to specify a desired goal or as an output modality to visualize robot intent or internal state. A recent paper [1] explores how AR can be combined with MoveIt to specify basic pick and place tasks, and offer previews of plans computed with MoveIt. By specifying only high-level goals, the user doesn't have to worry about _how_ a robot should achieve some goals, and, instead, can rely on MoveIt to provide solution paths. In the image above a user places virtual cans in desired locations. The inset figure shows the user’s point of view. The robot takes this input and computes a plan to place real cans in those locations.
```

Comment:
We did something very similar in

    Dennis Krupke, Frank Steinicke, Paul Lubos, Yannick Jonetzko, Michael Görner and Jianwei Zhang,
    Comparison of Multimodal Heading and Pointing Gestures for Co-Located Mixed Reality Human-Robot Interaction,
    International Conference on Intelligent Robots and Systems, IROS 2018, Madrid, Spain.

I would be grateful if you add it here.

Reply:
> Nice idea.
> 
> Is there a way to link to the references from within the text?
> It's probably also a good idea to link the respective doi pages.

If we used Jekyll-scholar it'd be easy, but github doesn't support this. 
 
> One other thing that comes to mind is @pradeepr-roboticist's papers cited in [ros-planning/moveit#2097](https://github.com/ros-planning/moveit/issues/2097) . I did not look further into it though.

There seems to be enough there for a whole separate post. Perhaps we could have a "guest post" in the future?